### DISCUSSION TITLE
Switch between PHP versions

### DISCUSSION THREAD

**Discussion by DarkCorner-IT** [leokhoa/laragon#523] [General]
I have to test websites that need to use different PHP versions.
So each host must have its own configuration.
Furthermore, I also need to change the PHP configuration of a host, switching from one version to another just for testing.
For example going from PHP version 8 to version 7 and then back again to version 8.
On my web server I can do the PHP version switch without problems, but I don't understand if it can be done with Laragon, or how to do it.
Now I'm working on Win 10 and Win 11.
Thanks in advance.

**Comment by Pen-y-Fan** [comment]
See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/)

**Reply by Pen-y-Fan** [reply]
If you need two instances running at the same time see #433
